internal/commands/root.go

```diff
@@ -94,8 +94,8 @@ type credentialOptions struct {
 }
 
 func (o *credentialOptions) addFlags(flags *pflag.FlagSet) {
-	flags.StringVar(&o.targetContext, "target-context", "", "Context on which the application is executed")
-	flags.StringArrayVarP(&o.credentialsets, "credential-set", "c", []string{}, "Use a credentialset (either a YAML file, or a credential set present in the credential store)")
+	flags.StringVar(&o.targetContext, "target-context", "", "Context on which the application is installed")
+	flags.StringArrayVar(&o.credentialsets, "credential-set", []string{}, "Use a YAML file containing a credential set or a credential set present in the credential store")
```

Comment:
Is the credential-store still supported ?

Reply:
It is (as otherwise we can't deploy generic CNABs) but it needs some improvements. It should accept a YAML file but right now it only looks inside the credential store for the target context.